Shapes.java

```diff
@@ -1,20 +1,23 @@
-import java.util.*;
-
-public class Shapes{
-  public static void main(String[] args) {
+import java.util.Scanner;
+import java.lang.*;
+       public class shapes{
```

Comment:
You were right before - the name of the class should be capitalized, so `public class Shapes` should be the line with the name of the class.